addons/account/models/account_move.py

```diff
@@ -4633,7 +4659,7 @@ def _generate_pdf_and_send_invoice(self, template, force_synchronous=True, allow
         """
         composer_vals = self._get_pdf_and_send_invoice_vals(template, **kwargs)
         composer = self.env['account.move.send'].create(composer_vals)
-        return composer.action_send_and_print(force_synchronous=force_synchronous, allow_fallback_pdf=allow_fallback_pdf, bypass_download=bypass_download)
+        composer.action_send_and_print(force_synchronous=force_synchronous, allow_fallback_pdf=allow_fallback_pdf, bypass_download=bypass_download)
```

Comment:
Can't we just have the opposite of `force_synchronous`, which forces the use of the CRON?

Reply:
Something is done like that in https://github.com/odoo/odoo/pull/214390 , but I do not want users to have to wait again if they go through the Send button on the invoice.